centos-7.template

```diff
@@ -117,9 +118,10 @@ cat > cert-gen.sh.base64 << EOF
 ${cert_gen}
 EOF
 
-base64 -d < cert-gen.sh.base64 > cert-gen.sh
-chmod +x cert-gen.sh
-mv cert-gen.sh /opt
+base64 -d < cert-gen.sh.base64 > minishift-cert-gen
+rm cert-gen.sh.base64
```

Comment:
Do we need to put `rm -fr ...`? It should not break if user-interaction required.